SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTestWorkerDelegationComponent.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialCommonTypes.h"
+#include "SpatialFunctionalTestWorkerDelegationComponent.generated.h"
+
+/*
+ * Actor Component that allows you to delegate its Actor to a specific Server Worker.
```

Comment:
Maybe I would add, that this is for use in tests only, so when it comes up in the editor, people don't mistakenly use it for unintended purposes.